Sources/TensorFlow/Core/LazyTensorOperation.swift

```diff
@@ -162,6 +162,7 @@ class LazyTensorOperation: TensorOperation {
     let outputCount: Int
     var inputs: [Input]
     var attributes: [String: Attribute]
+    var device: String?
```

Comment:
Since this is not a device but a device name, how about calling it `deviceName` instead?

Reply:
Done.